contribs/src/main/java/com/netflix/conductor/contribs/http/HttpTask.java

```diff
@@ -159,13 +159,14 @@ protected HttpResponse httpCall(Input input) throws Exception {
 			OAuthSecrets secrets = new OAuthSecrets().consumerSecret(input.oauthConsumerSecret);
 			client.addFilter(new OAuthClientFilter(client.getProviders(), params, secrets));
 		}
-
-		Builder builder = client.resource(input.uri).type(MediaType.APPLICATION_JSON);
+		final String contentType = "content-type";
```

Comment:
What about different variations?  e.g. `Content-Type` ?

Reply:
http headers are case insensitive so header hashmap should have all keys in same case.
If I change the setter of header in Input class all header will be lower case.
public void setHeaders(Map<String, Object> headers) {
			 for (String key : headers.keySet()) {
		            this.headers.put(key.toLowerCase(),headers.get(key));
		        }
		}

Reply:
@nsiitk @v1r3n according to the Conductor [docs](https://netflix.github.io/conductor/metadata/systask/#http), there is a distinct parameter `contentType` that should be used for setting the content type of the request. 

I created a similar PR to use that parameter, see PR #391

Reply:
@nsiitk as @picaron mentioned, `contentType` is a distinct parameter and should be used to override the contentType.